components/library/opal/Makefile

```diff
@@ -82,3 +82,12 @@ REQUIRED_PACKAGES += system/library
 REQUIRED_PACKAGES += $(GXX_RUNTIME_PKG)
 REQUIRED_PACKAGES += $(GCC_RUNTIME_PKG)
 REQUIRED_PACKAGES += system/library/math
+# Auto-generated dependencies
+REQUIRED_PACKAGES += $(GCC_RUNTIME_PKG)
+REQUIRED_PACKAGES += $(GXX_RUNTIME_PKG)
+REQUIRED_PACKAGES += codec/libtheora
+REQUIRED_PACKAGES += codec/speex
+REQUIRED_PACKAGES += library/desktop/ptlib
+REQUIRED_PACKAGES += library/speexdsp
+REQUIRED_PACKAGES += system/library
+REQUIRED_PACKAGES += system/library/math
```

Comment:
Please cleanup duplicates (basically all `REQUIRED_PACKAGES` lines above the `# Auto-generated dependencies` comment line should be removed).